frontend/src/metabase/lib/pagination.ts

```diff
@@ -0,0 +1,19 @@
+import type { PaginationInput, PaginationRequest } from "metabase-types/api";
+
+const DEFAULT_PAGE = 0;
+const DEFAULT_PAGE_SIZE = 50;
+
+export const getPaginationRequestParams = (
```

Comment:
I copied the code and put it in the description of this PR, but here's the link as well:
https://github.com/metabase/metabase/pull/41452/files#diff-2fc03e43ac8dcabe663179f3473a1db3ab319f96124ddb746c0fa4b73403792dR25-R26

You can't only pass `pageSize` and call it a day because there needs to be a control mechanism somewhere, and that's why we have the `page`.
https://github.com/metabase/metabase/pull/41452/files#diff-2fc03e43ac8dcabe663179f3473a1db3ab319f96124ddb746c0fa4b73403792dR57-R58

We could, in theory, still deal with the page and then do calculations directly in the file rather than using a helper.
So instead of this
```js
const [page, setPage] = useState(0);
const pagination = getPaginationRequestParams({ page });
const { data: tasksData } = useListTasksQuery(pagination);
```

Do this
```js
const [page, setPage] = useState(0);
const { data: tasksData } = useListTasksQuery({ limit: 50, offset: page * 50 });
```
We can either store `50` as the default limit, and obviously we could pass some other number in places where it's needed.

Reply:
I like this second example and am ok with removing the helper for now.
You're right - we don't have too many places where we have pagination.

The downside is: if the backend pagination implementation changes, we have to update the FE logic in a dozen different places instead of just one.

Reply:
The bottom line is that the helper isn't a bad thing since it allows composition.
I'm fine with having it but I wanted to point out that we won't loose much if we live without it.

Reply:
The helper is gone. Poof.